website/content/docs/api-gateway.mdx

```diff
@@ -0,0 +1,171 @@
+---
+layout: docs
+page_title: API Gateway
+description: Using Consul API gateway functionality
+---
+
+# Consul API Gateway
+
+This topic describes how to use the Consul API Gateway module, which enables applications and services running in a datacenter to be accessed by network clients outside the datacenter. This type of network traffic is commonly referred to as "north-south" network traffic.
+
+## Introduction
+
+Consul API Gateway is an implementation of the Kubernetes Gateway [API Specification](https://gateway-api.sigs.k8s.io/). The specification defines a set of custom resource definitions (CRD) that can create logical gateways and routes.
+
+## Requirements
+
+Your datacenter must meet the following requirements prior to configuring the Consul API Gateway:
+
+- A Kubernetes cluster must be running
+- Consul 1.11.1+
+
+## Installation
+
+1. Create a values file for your Consul server agents that contains the following parameters:
+
+   ```yaml
+   global:
+     name: consul
+     image: 'hashicorp/consul:1.11.1'
+     tls:
+       enabled: true
+   connectInject:
+     enabled: true
+   controller:
+     enabled: true
+   ```
+
+1. Install Consul API Gateway using the standard Consul Helm chart and specify the custom values file.
+
+```shell-session
+$ helm install consul hashicorp/consul --version 0.39.0 -f values.yaml
+```
+
+The following components will be installed: - Gateway controller - CRDs required by the Kubernetes Gateway API specification - `kustomize` manifests for completing the installation
+
+1. After `helm` installs Consul API Gateway packages, issue the following commands to apply the API gateway to your Kubernetes cluster.
+
+```shell-session
+$ kubectl apply -k "github.com/hashicorp/consul-api-gateway/config/crd?ref=v0.1.0-techpreview"
+$ kubectl apply -k "github.com/hashicorp/consul-api-gateway/config?ref=v0.1.0-techpreview"
+```
+
+## Usage
+
+1. Verify that the [requirements](#requirements) have been met.
+1. Verify that the Consul API Gateway software has been installed and applied (see [Installation](#installation)).
+1. Configure the gateway, listener(s), and route(s) as described in [Configuration](#configuration)
```

Comment:
I think that's a good call for tutorials where the sequence has to be precise. The template I'm trying to institute is to describe the procedure as concisely as possible and link to large sets of details--almost like an appendix--so that readers flow isn't interrupted as they're taking in a potentially new concept.